Classes/Service/ExtensionService.php

```diff
@@ -79,4 +86,14 @@ public function isComposerStoragePath(string $path): bool
         }
         return false;
     }
+
+    public function isStoragePathConfigured(): bool
+    {
+        if (Environment::isComposerMode() && count($this->resolveStoragePaths()) === 0) {
```

Comment:
simplify to

`return !Environment::isComposerMode() || count($this->resolveStoragePaths());`

Reply:
count(arr) returns int not boolean so I used count(arr) > 0

Reply:
> count(arr) returns int not boolean

Does not really matter. This conversion is done automatically anyways. 0 = false is the same for every programming language I know. ;-)

Reply:
We could even skip the count as in PHP an empty array is `false` too.